packages/kbn-es/src/install/install_archive.ts

```diff
@@ -62,9 +62,11 @@ export async function installArchive(archive: string, options?: InstallArchiveOp
   });
   log.info('extracted to %s', chalk.bold(installPath));
 
-  const tmpdir = path.resolve(installPath, 'ES_TMPDIR');
-  fs.mkdirSync(tmpdir, { recursive: true });
-  log.info('created %s', chalk.bold(tmpdir));
+  if (process.env.FTR_DISABLE_ES_TMPDIR !== 'true') {
```

Comment:
@Ikuni17 could you add a small description when `FTR_DISABLE_ES_TMPDIR` must be set to true and how it changes the behaviour.

Reply:
Added 405f845b2cfd3b902ada4c2cc9de1cc4ac36d88f